pkg/utils/eventrecorder/eventrecorder.go

```diff
@@ -108,21 +106,3 @@ func findMyPod(cachedK8SClient client.Client) (corev1.Pod, error) {
 	}
 	return pod, err
 }
-
-// SendNodeEvent sends an event regarding pod object and related to node object
-func (e *EventRecorder) SendNodeEvent(eventType, reason, action, message string) error {
```

Comment:
I think `findMyNode` can be removed here as well, as only `SendNodeEvent` uses `e.hostNode`

Reply:
Thanks, Jeff. Let me check on the function. Also will confirm the tests.

Reply:
Tests focused on pod_eni was passed.
```
[AfterSuite] PASSED [173.059 seconds]
------------------------------

Ran 1 of 6 Specs in 614.365 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 5 Skipped
PASS | FOCUSED

Ginkgo ran 1 suite in 10m23.396413883s
Test Suite Passed
```